client-cli/src/command.rs

```diff
@@ -853,7 +853,7 @@ impl Command {
                 } => {
                     if let Some(ref mut pb) = progress_bar {
                         if current_block_height == final_block_height {
-                            pb.finish_println("Synchronization complete!\n");
+                            pb.finish_print("Synchronization complete!");
```

Comment:
why no new lines?

Reply:
println + \n makes double display of the same message , so changed to like this